### ISSUE TITLE
Enable aarch64 support in Ubuntu PPA

### ISSUE THREAD

**Issue by trevormoseyeatstreet** [drakkan/sftpgo#219]
Please enable the aarch64 target in the Ubuntu PPA

**Comment by drakkan** [comment]
Hi,

I'll evaluate this request for future releases but it is no so easy: the PPA uses the binary release for amd64, and we currently don't have a binary release for aarch64.

However the docker image is available for aarch64 too

**Comment by drakkan** [comment]
Hi,

I just released 1.2.0 and I updated the build system to produce binaries for aarch64 too. You can download the produced deb/rpm [here](https://github.com/drakkan/sftpgo/releases/tag/v1.2.0).

I had some troubles adding arm64 support to the Ubuntu PPA, mainly related to package version. 

The sources for the PPA package are [here](https://github.com/drakkan/sftpgo/tree/master/pkgs/debian). I think I need to add an arm64 specific install file, something like this:

```
$ cat sftpgo.install.arm64 
arm64/sftpgo usr/bin
examples/rest-api-cli/sftpgo_api_cli usr/bin
sftpgo.json etc/sftpgo
init/sftpgo.service lib/systemd/system
bash_completion/sftpgo usr/share/bash-completion/completions
man/man1/* usr/share/man/man1
templates usr/share/sftpgo
static usr/share/sftpgo
``` 

and then change Architecture to `any` inside the control file.

I'll retry when I'll release the next version, meantime I'll appreciate if you could test the deb package,
 
thank you

**Comment by drakkan** [comment]
The Ubuntu PPA now supports amd64,arm64 and ppc64el, thank you
